Makefile

```diff
@@ -20,3 +20,7 @@ postgres:
 pygments:
 	out=$$(edb gen-meta-grammars edgeql) && \
 		echo "$$out" > edb/edgeql/pygments/meta.py
+
+
+update:
```

Comment:
I'd call this "build" and make the default target.

Reply:
+1.

Reply:
@ambv, please update the target here.